.gitignore

```diff
@@ -554,6 +576,11 @@ build-iPhoneSimulator/
 
 ### VisualStudioCode ###
 .vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
```

Comment:
I understand that this is part of the template but I think we don't need these excludes. E.g. I don't think it's necessary to commit my VS Code workspace settings for a public example app.